demosplan/DemosPlanCoreBundle/Logic/OzgKeycloakUserDataMapper.php

```diff
@@ -109,40 +110,42 @@ public function __construct(
     }
 
     /**
+     * Maps incoming data to dplan:user.
+     * Creates
+     *
      * @throws CustomerNotFoundException
      * @throws Exception
      */
-    public function handleKeycloakData(OzgKeycloakResponseValueObject $ozgKeycloakResponseValueObject): User
+    public function mapUserData(KeycloakUserDataInterface $ozgKeycloakUserData): User
     {
-        $this->ozgKeycloakResponseValueObject = $ozgKeycloakResponseValueObject;
-        // 1 get Desired Roles
-        $requestedRoles = $this->mapKeycloakRoleNamesToDplanRoles();
-        // 2 handle Organisation / load it / update it / create it --- handle special case CITIZEN
-        $requestedOrga = $this->getOrgaAndHandleRequestedOrgaData($requestedRoles);
-        // 3 handle user / load it / update it / create it / and add User to Orga and Department
+        $this->ozgKeycloakUserData = $ozgKeycloakUserData;
 
+        $requestedRoles = $this->mapUserRoleData();
+        $requestedOrganisation = $this->mapUserOrganisationData($requestedRoles);
+        // 3 handle user / load it / update it / create it / and add User to Orga and Department
```

Comment:
This seems a bit weird as 1 & 2 got lost

Reply:
I (re)moved these.
First was "// 1 get Desired Roles" which is clear because of the meanwhile named method "mapUserRoleData()"
Second was moved to docblock of mapUserOrganisationData(), but wasn't finished. I fixed it with:
0356b8ee617d346ebfef6632d0f0d9f1e7145808

Reply:
I tried to make it more clear by rename methods and replace and adjust the existing comments
1ddae2061b9d8d8bf5c16c41b592f65d55a41ef6

Reply:
thnk you